script.js

```diff
@@ -0,0 +1,30 @@
+//Declare and initilize variables
+let howOldAreYou;
```

Comment:
Normally there's no reason to initialize variables in advance, you can simply initialize them down the line where you use it (line 15)